bin/git-theta-filter

```diff
@@ -59,26 +68,43 @@ def clean(args):
     checkpoint_handler = checkpoints.get_checkpoint_handler()
     model_checkpoint = checkpoint_handler.from_file(sys.stdin.buffer)
 
+    update_serializer = params.get_update_serializer()
     prev_metadata = metadata.Metadata.from_commit(repo, args.file, "HEAD").flatten()
 
     async def _clean(param_keys, new_param):
         logging.debug(f"Cleaning {'/'.join(param_keys)}")
         param_metadata = prev_metadata.get(param_keys)
         new_tensor_metadata = metadata.TensorMetadata.from_tensor(new_param)
+
         # If the parameter tensor has not changed, just keep the metadata the same
-        # TODO: currently checks if rounded parameter hash has changed. Look into better parameter equality checks.
-        if param_metadata and param_metadata.tensor_metadata == new_tensor_metadata:
-            return param_keys, param_metadata
+        # TODO: Currently uses some magic numbers for LSH distance thresholds.
```

Comment:
Moved the magic numbers into `utils.EnvVarConstants` so that they are configurable. Also to avoid having to make a getter for every environment variable that defaults to the correct value, I just built that into the `EnvVarConstants` class.